yext/analytics
# expose endpoint field in ChatAnalyticsConfig

To allow for endpoints from qa/dev environment, such as chat preview panel in storm, this `endpoint` is added to override the URLs to send requests to.


J=CLIP-306
TEST=manual&auto

- see that i'm able to override the endpoint to a dev endpoint and sent a successful request
- see that jest tests passed
